Lib9c/Action/TransferAsset.cs

```diff
@@ -9,21 +9,34 @@
 using System.Linq;
 using System.Runtime.Serialization;
 using Lib9c.Abstractions;
+using Nekoyume.Helper;
 using Nekoyume.Model;
 using Serilog;
 
 namespace Nekoyume.Action
 {
     /// <summary>
     /// Hard forked at https://github.com/planetarium/lib9c/pull/636
-    /// Updated at https://github.com/planetarium/lib9c/pull/957
+    /// Updated at https://github.com/planetarium/lib9c/pull/1718
     /// </summary>
     [Serializable]
     [ActionType("transfer_asset3")]
     public class TransferAsset : ActionBase, ISerializable, ITransferAsset, ITransferAssetV1
     {
         private const int MemoMaxLength = 80;
 
+        // FIXME justify this policy.
+        public const long CrystalTransferringRestrictionStartIndex = 6_220_000L;
+
+        // FIXME justify this policy.
+        public static readonly IReadOnlyList<Address> AllowedCrystalTransferList = new List<Address>
```

Comment:
```suggestion
        public static readonly ISet<Address> AllowedCrystalTransfers = new List<Address>
```

Sorry for the late & trivial checking. I think we don't need to keep the order. 😅